src/brouwers/shop/translation.py

```diff
@@ -0,0 +1,7 @@
+from modeltranslation.translator import register, TranslationOptions
+from .models import Category
+
+
+@register(Category)
+class CategoryTranslationOptions(TranslationOptions):
+    fields = ('name', 'slug', 'seo_keyword', 'enabled')
```

Comment:
Hrm, `enabled` is not user facing, I think it will just control if the category is visible or not in the frontend. So this doesn't have to be translatable (also doesn't make sense to have a translatable BooleanField).

Reply:
Removed.